iroha/src/sumeragi.rs

```diff
@@ -1839,7 +1839,7 @@ pub mod config {
 
 #[cfg(test)]
 mod tests {
-    #![allow(clippy::restriction)]
+    #![allow(clippy::restriction, clippy::too_many_lines)]
```

Comment:
Nothing has changed below, why add `too_many_lines` allow lint?

Reply:
Clippy is weird. May be macro has triggered it... But there functions >100loc